write-fonts/src/tables/gvar.rs

```diff
@@ -417,6 +429,11 @@ pub struct GlyphVariationData {
     // optional; present if multiple variations have the same point numbers
     shared_point_numbers: Option<PackedPointNumbers>,
     per_tuple_data: Vec<GlyphTupleVariationData>,
+    /// calculated length required to store this data
+    ///
+    /// we compute this once up front because we need to know it in a bunch
+    /// of different places
+    length: u32,
```

Comment:
usize seems the more natural type, I suppose you eventually want to assign it somewhere u32. Perhaps note why it's u32 here?